client/state/jetpack/credentials/test/reducer.js

```diff
@@ -0,0 +1,99 @@
+import { markCredentialsAsInvalid, markCredentialsAsValid, testCredentials } from '../actions';
+import { testRequestStatus } from '../reducer';
+
+describe( 'reducer', () => {
+	describe( 'testRequestStatus', () => {
```

Comment:
Definitely not a blocker for this PR, but ideally, this reducer should have tests for the rest of its subreducers.